src/main/java/com/beis/subsidy/award/transperancy/dbpublishingservice/util/AwardUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.beis.subsidy.award.transperancy.dbpublishingservice.util;
+
+public class AwardUtils {
+
+    public static boolean validateCompanyNumber(String companyNumber) {
+
+        if(companyNumber!=null && (companyNumber.length()!=8 || !companyNumber.matches("[A-Za-z0-9]+"))) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
Should probably return true if the company number is of a correct format, this returns false if its correct.
However, even if we are returning false if its correct, the expression should be:
`if(companyNumber == null || companyNumber.length() != 8 || !companyNumber.matches("[A-Za-z0-9]+"))
{
    return true
}
    `
Currently, if the companyNumber was null it would return false (which would mean it would pass validation)

Returning just the boolean expression would be clearer:
`return companyNumber != null && companyNumber.length() == 8 && companyNumber.toLowerCase().matches("[a-z0-9]+");`

This would then need updates in the addawardservice and bulkuploadawardservice as the return value would be opposite to what is currently there